frontend/src/scenes/pipeline/Overview.tsx

```diff
@@ -1,161 +1,24 @@
-    </LemonCard>
-)
-
-const PipelineStepTransformation = ({ transformation }: { transformation: Transformation }): JSX.Element => {
-    const { sortedEnabledTransformations } = useValues(pipelineTransformationsLogic)
-
-    return (
-        <PipelineStep
-            order={sortedEnabledTransformations.findIndex((pc) => pc.id === transformation.id) + 1}
-            name={transformation.name}
-            to={urls.pipelineNode(PipelineStage.Transformation, transformation.id, PipelineNodeTab.Configuration)}
-            description={transformation.description}
-            headerInfo={
-                <>
-                    <div className="mr-1">
-                        <AppMetricSparkLine pipelineNode={transformation} />
-                    </div>
-                    <More overlay={<TransformationsMoreOverlay transformation={transformation} inOverview />} />
-                </>
-            }
-            plugin={transformation.plugin}
-        />
-    )
-}
-
-const PipelineStepDestination = ({ destination }: { destination: Destination }): JSX.Element => {
-    return (
-        <PipelineStep
-            name={destination.name}
-            to={urls.pipelineNode(PipelineStage.Destination, destination.id, PipelineNodeTab.Configuration)}
-            description={destination.description}
-            headerInfo={
-                <>
-                    <div className="mr-1">
-                        <AppMetricSparkLine pipelineNode={destination} />
-                    </div>
-                    <More overlay={<DestinationMoreOverlay destination={destination} inOverview />} />
-                </>
-            }
-            additionalInfo={
-                <div className="flex gap-1">
-                    <LemonTag type="primary">{humanFriendlyFrequencyName(destination.interval)}</LemonTag>
-                </div>
-            }
-            plugin={destination.backend === PipelineBackend.Plugin ? destination.plugin : undefined}
-        />
-    )
-}
+import { DestinationsTable } from './Destinations'
+import { TransformationsTable } from './Transformations'
 
 export function Overview(): JSX.Element {
-    const { transformations, destinations, transformationsLoading, destinationsLoading } =
-        useValues(pipelineOverviewLogic)
-
     return (
         <div>
             <h2 className="mt-4">Transformations</h2>
-            <div className="grid grid-cols-3 gap-4">
-                {transformationsLoading ? (
-                    <PipelineStepSkeleton />
-                ) : (
-                    transformations &&
-                    transformations
-                        .filter((t) => t.enabled)
-                        .map((t) => <PipelineStepTransformation key={t.id} transformation={t} />)
-                )}
-            </div>
-
+            Showing only enabled, go to <Link to={urls.pipeline(PipelineTab.Transformations)}>
+                Transformations tab
+            </Link>{' '}
+            to see all.
```

Comment:
I think we could spark more joy by just having the title be "Active transformations" and then an inline "View all" button, but this does work